docs/getting-started/first-source.md

```diff
@@ -52,14 +76,15 @@ We can use the CloudEvents Player to send and receive CloudEvents. If you open t
 
     For more information on the CloudEvents Specification, check out the [CloudEvents Spec](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md){target=_blank}.
 
-Fill out the form with whatever you data you would like to (make sure your Event Source does not contain any spaces) and hit the "SEND EVENT" button.
+Fill out the form with whatever you data you would like to **(make sure your Event Source does not contain any spaces)** and hit the "SEND EVENT" button.
 
 
-![CloudEvents Player Send](https://user-images.githubusercontent.com/16281246/116407683-04448d80-a800-11eb-9283-ba86fb259053.png)
+![CloudEvents Player Send](../images/event_sent.png)
 
 
-!!! tip "Tip: Clicking the :fontawesome-solid-envelope: will show you the CloudEvent as the `Broker` sees it."
+??? tip "Clicking the :fontawesome-solid-envelope: will show you the CloudEvent as the Broker sees it."
```

Comment:
??? tip "Clicking the :fontawesome-solid-envelope: shows you the CloudEvent as the Broker sees it."